src/metal.js

```diff
@@ -1,3 +1,21 @@
-// Add Backbone.Metal extensions here
-// 1. Marionette.Class
-// 2. Marionette.Error and Marionette.Deprecate
+// Missing Metal Items
+// 1. Marionette.Class parity
+// 2. Marionette.Deprecate
+
+Marionette.Class = Metal.Class.extend({});
+
+function classify(obj) {
+  return Marionette.Class.extend(
+    _.extend({constructor: obj}, _.omit(obj.prototype, _.keys(Backbone.Events))),
+    _.omit(obj, _.keys(Marionette.Class))
+  );
```

Comment:
I know there's not much going on in this classify function, but all the various extends and omits are hard to keep straight, maybe a bit more wordy for some clarity?
```js
var initKeys = ['constructor','initialize'];
var eventKeys = _.keys(Backbone.Events);
var classKeys = _.keys(Marionette.Class);

function classify(obj) {
  var protoProps = _.extend({constructor: obj}, _.omit(obj.prototype, eventKeys, initKeys));
  var staticProps = _.omit(obj, classKeys, initKeys);

  return Marionette.Class.extend(protoProps, staticProps);
}
```